### ISSUE TITLE
Video - accessibility

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7462]
The following should be done to make the video component more accessible:
- Improve aria-label on menu control
- Screenreader user can’t use left and right arrow keys to forward or rewind the video
- Focus indicator isn’t visible in the menu
- Focus indicator gets cut off when controls are below
- Controls position over, can’t see captions while controls are visible

In our examples we should show videos with the following:
- Description of visual information
- Captions/subtitles
- Transcripts (text version the speech and non-speech audio information)

Bonus features:
- Change speed of video
- Set how captions are displayed (text style, size, colors, position)
- Create a pattern for displaying audio description

Relevant WCAG rules and links:
[1.2.1 Audio-only and Video-only (Prerecorded) Level A](https://www.w3.org/WAI/WCAG22/Understanding/audio-only-and-video-only-prerecorded.html)
[1.2.2 Captions (Prerecorded) Level A](https://www.w3.org/WAI/WCAG22/Understanding/captions-prerecorded.html)
[1.2.3 Audio Description or Media Alternative (Prerecorded) Level A](https://www.w3.org/WAI/WCAG22/Understanding/audio-description-or-media-alternative-prerecorded.html)
[1.2.5 Audio Description (Prerecorded) Level AA](https://www.w3.org/WAI/WCAG22/Understanding/audio-description-prerecorded.html)

https://www.w3.org/WAI/media/av/
